.travis.yml

```
@@ -8,7 +8,7 @@ install:
     - pip install tox
 
 script:
-    - tox
+  - tox
```

This change is inconsistent with the rest of the file.
```suggestion
    - tox
```